capstone/src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -45,7 +45,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String ChannelId = request.getParameter("channelId");
     String videoID = request.getParameter("videoID");
```

Comment:
nit: `ChannelId` -> `channelId` and let's keep the `id` or `ID` in the parameters consistent

Reply:
addressed in "general formatting"